legion/engine/core/defaults/coremodule.hpp

```diff
@@ -30,6 +31,8 @@ namespace legion::core
             reportComponentType<position>();
             reportComponentType<rotation>();
             reportComponentType<scale>();
+            reportComponentType<scenemanagement::scene>();
+            reportSystem < ::legion::core::scenemanagement::SceneManager > ();
```

Comment:
wait you already report in core module, so why does scene module exist?

Reply:
scene is a component?